ubi8-py36/requirements.txt

```diff
@@ -1,5 +1,4 @@
-#
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.8
```

Comment:
seems like a wrong compile

Reply:
aaa thanks for spotting this :) I copied wrong one

Reply:
updates :)